modules/gdscript/tests/scripts/parser/errors/annotation_unrecognized.gd

```diff
@@ -0,0 +1,2 @@
+@hello_world
+var test
```

Comment:
As far as I know, the file must have a `test()` function.

Reply:
Kinda assumed that it wasn't necessary for parser errors since they don't actually run iirc. But yeah better to have it consistent.